app/components/Autopay/AutopayCardView.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex } from 'rebass'
+import { Card, Heading } from 'components/UI'
+
+const CardWithBg = styled(Card)`
+  position: relative;
+  background-image: url(${props => props.src});
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  height: 300px;
+  width: 195px;
+  border-radius: 40px;
```

Comment:
Rebass actually has a `radii` property that can be set in the theme that we can use to set predefined reusable values for borderRadius prop. I will probably leave this as it is for this PR but we can follow up with a refactor PR that extracts our border radiuses to the theme so that we have some consistency and more easy reusability.